src/pybricks/hubs.py

```diff
@@ -7,9 +7,10 @@
                       LightMatrix as _LightMatrix, IMU as _IMU,
                       System as _System,
                       SimpleAccelerometer as _SimpleAccelerometer)
+from .geometry import Axis as _Axis
+from .ev3dev._speaker import Speaker as _EV3Speaker
```

Comment:
It would be nice to keep these in alphabetical order.

Reply:
Yep, no problem. Forgot to hit the import optimisation shortcut. Fixed now.